spec/acceptance/signature_webhook_spec.rb

```diff
@@ -47,5 +47,12 @@ class {'r10k::webhook':
         expect(r.exit_code).to eq(0)
       end
     end
+    it 'should respond with a 200 with status=success in JSON format on /heartbeat' do
```

Comment:
This does not actually check for a 200 HTTP status code. You could do that in a similar to fashion as done here with `curl -I -X GET`